pkg/mounter/safe_mounter_windows.go

```diff
@@ -68,6 +71,9 @@ func normalizeWindowsPath(path string) string {
 }
 
 func (mounter *csiProxyMounter) SMBMount(source, target, fsType string, mountOptions, sensitiveMountOptions []string) error {
+	mutex.Lock()
```

Comment:
this lock is too big, is it possible to lock like this, lock based on root mapping path string?

https://github.com/kubernetes-csi/csi-driver-smb/blob/a408bba9ff8dc10e3aebef04d855f6b81c61318a/pkg/smb/nodeserver.go#L153-L156

Reply:
I've removed global mutex at SMBMount/SMBUnmount.
After that I've add locks base on mappingPath for inc/dec operations with ref counts.